pyaedt/application/Analysis3D.py

```diff
@@ -291,6 +291,65 @@ def set_source_context(self, sources, number_of_modes=1):
         self.osolution.SetSourceContexts(contexts)
         return True
 
+    @aedt_exception_handler
+    def assign_material(self, obj, mat):
+        """Assign a material to one or more objects.
+
+        Parameters
+        ----------
+        obj : str, list
+            One or more objects to assign materials to.
+        mat : str
+            Material to assign. If this material is not present, it will be
+            created.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
```

Comment:
Let's get in the habit of adding an `Example` section for all our methods as we update the API.

Reply:
Done